app/src/main/java/com/concordium/wallet/core/config/EnvironmentConfiguration.kt

```diff
@@ -0,0 +1,8 @@
+package com.concordium.wallet.core.config
+
+object EnvironmentConfiguration {
+    const val baseUrl = "https://wallet-proxy.mainnet.concordium.software"
```

Comment:
Why is this hardcoded to mainnet?

Reply:
removed it, for now, and will update the configuration to all flavours with separate PR/task. 
This is base class that will expand when I move environment configuration from gradle->EnvironmentConfiguration  object, which is defined per flavour package.